core/errors/infer.h

```diff
@@ -1,6 +1,7 @@
 #ifndef SORBET_CORE_ERRORS_INFER_H
 #define SORBET_CORE_ERRORS_INFER_H
 #include "core/Error.h"
+#include "core/SymbolRef.h"
```

Comment:
Does this need to be
```suggestion
#include "core/TypePtr.h"
```

?